src/ValueObject/GalleryValueObject.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme\ValueObject;
+
+/**
+ * Handle information about a media item.
+ */
+class GalleryValueObject extends ValueObjectBase {
```

Comment:
This value object represent a single item in a gallery and not the gallery itself, wouldn't it be clearer to call it `GalleryItamValueObject`? Also the class comment will need to be updated.

Reply:
Resolved.